src/address.rs

```diff
@@ -38,3 +38,21 @@ impl Address {
         &self.pk_d
     }
 }
+
+/// Generators for property testing.
+#[cfg(any(test, feature = "test-dependencies"))]
+pub mod testing {
+    use proptest::prelude::*;
+
+    use crate::keys::{testing::arb_spending_key, FullViewingKey};
+
+    use super::Address;
+
+    prop_compose! {
+        /// Generate an arbitrary random seed
```

Comment:
```suggestion
        /// Generates an arbitrary payment address.
```

The summary line for Rust doc comments should be [written in third person singular present indicative form](https://doc.rust-lang.org/1.0.0/style/style/comments.html#sentence-structure).